workshop/exercise-4/README.md

```diff
@@ -115,7 +105,7 @@ You can read more about how [Istio mixer enables telemetry reporting](https://is
      $(kubectl -n istio-system get pod -l app=prometheus -o jsonpath='{.items[0].metadata.name}') \
      9090:9090
    ```
-2. Browse to http://localhost:9090/graph, and in the “Expression” input box, enter: istio_request_count. Click Execute.
+2. Browse to http://localhost:9090/graph, and in the “Expression” input box, enter: istio_request_byte_count. Click Execute.
```

Comment:
@irisdingbj q on this change, is istio_request_count no longer avail?  It seems to be avail when I tried it yesterday.